docs/resources/mnq_sqs_queue.md

```diff
@@ -0,0 +1,84 @@
+---
+subcategory: "Messaging and Queuing"
+page_title: "Scaleway: scaleway_mnq_sqs_queue"
+---
+
+# scaleway_mnq_sqs_queue
+
+Creates and manages Scaleway Messaging and queuing SQS Queues.
+For further information please check
+our [documentation](https://www.scaleway.com/en/docs/serverless/messaging/how-to/create-manage-queues/)
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_mnq_sqs" "main" {}
+
+resource scaleway_mnq_sqs_credentials main {
+  project_id = scaleway_mnq_sqs.main.project_id
+  name = "sqs-credentials"
+
+  permissions {
+    can_manage = false
+    can_receive = true
+    can_publish = false
+  }
+}
+
+resource scaleway_mnq_sqs_queue main {
+  project_id = scaleway_mnq_sqs.main.project_id
+  name = "my-queue"
+  endpoint = scaleway_mnq_sqs.main.endpoint
```

Comment:
Could endpoint be removed? I'm not sure to understand why I need to specify it

Reply:
Added a test without endpoint specified